Resources/input/fsh/ISiKKontaktGesundheitseinrichtung.fsh

```diff
@@ -208,3 +208,44 @@ Invariant: ISiK-enc-8
 Description: "Die Rolle der assoziierten Diagnose(n) darf nicht 'Billing' sein"
 Severity: #error
 Expression: "diagnosis.use.all(coding.code != 'billing')"
+
+
+Instance: Encounter-date-start
+InstanceOf: SearchParameter
+Usage: #definition
+* description = "The actual start date of the Encounter (Backport from R5)"
```

Comment:
Sollten wir hier ggf. hinzufügen "The parameter cannot repeat or have multiple values" gemäß http://hl7.org/fhir/R5/encounter-search.html#Encounter-date-start ?

Reply:
entsprechend umgesetzt